treeherder/webapp/api/performance_data.py

```diff
@@ -1447,6 +1675,31 @@ def _get_signatures_map(signatures, grouped_values, option_collection_map):
     8. Plot KDE with ISJ bandwidth to reduce smoothing
     """
 
+    @staticmethod
+    def _process_mann_whitney_task(
+        statistics_base_perf_data, statistics_new_perf_data, header, lower_is_better, common_result
+    ):
+        """
+        Process a single mann-whitney-u test task for parallel execution.
+        This is a static method so it can be pickled by multiprocessing.
+        """
+        # Suppress warnings during statistical processing to avoid cluttering output
+        with warnings.catch_warnings():
+            warnings.simplefilter("ignore")
+            new_stats = PerfCompareResults._process_stats(
+                statistics_base_perf_data,
+                statistics_new_perf_data,
+                header,
+                lower_is_better,
+                remove_outliers=False,
+            )
+
+        row_result = {
+            **common_result,
```

Comment:
Nit: Simplify the task method by only passing the base/new stat arrays, keep the large `common_result` in `_process_mann_whitney_u_version` and merge the results there (each result could have a task index to easily merge the results).